### PR TITLE
Fix for bug introduced with #8511

### PR DESCRIPTION
**Pull request by DaSchTour**
This was introduced with changes in #8511 
Here is the bad behaviour this causes http://codepen.io/DaSch/pen/YqgPBY

### CODE REVIEW

File: scss/grid/_flex-grid.scss [hunk 1]
```diff
@@ -99,7 +99,7 @@
   // fixes recent Chrome version not limiting child width
   // https://stackoverflow.com/questions/34934586/white-space-nowrap-and-flexbox-did-not-work-in-chrome
   @if $columns == null {
-    min-width: 0;
+    min-width: intial;
```
Comment (kball) [inline scss/grid/_flex-grid.scss]: looks like a typo - should this be initial?

**Comment by kball** [comment]
@ncoden based on your comments on @DaSchTour 's feedback on the other PR, I assume this seems like an appropriate fix to you?  So when the typo here is fixed we'll merge this... would either of you like to submit some visual tests around flexbox to help prevent this type of regression in the future?

**Comment by ncoden** [comment]
@kball It isn't a fix, it's just a revert. `initial` value is equel to removing the property.
I'll search for a fix that resolve my problem, without side effects.

**Comment by DaSchTour** [comment]
I hope we can find a fix very soon, so we can have this fixed in 6.2.2

**Comment by DaSchTour** [comment]
Is it this bug? https://github.com/philipwalton/flexbugs#1-minimum-content-sizing-of-flex-items-not-honored

**Comment by kball** [comment]
Merging this to include the reversion in the 6.2.2 rc, but it would be great if we get a fix for the original issue as well.

**Comment by DaSchTour** [comment]
@ncoden does min-width: min-content; maybe fix the problem? Autoprefixer adds -moz-min-content and -webkit-min-content

**Pull request merged**